adapter/internal/oasparser/model/open_api.go

```diff
@@ -181,6 +187,17 @@ func setSecuritySchemesOpenAPI(openAPI openapi3.Swagger) []SecurityScheme {
 	return securitySchemes
 }
 
+// getRequestBodyBufferConfig method returns a boolean value indicating whether a given API is configured to
+// pass request body to the enforcer or not.
+func getRequestBodyBufferConfig(vendorExtensions map[string]interface{}) bool {
+	if val, found := vendorExtensions[constants.XWso2PassRequestPayloadToEnforcer]; found {
+		if passerValue, ok := val.(bool); ok {
+			return passerValue
+		}
+	}
+	return true
```

Comment:
shouldn't we set false here?

Reply:
Ah got it since we check the the config's value prior to this. In that case, can't we read and assign the variable straightaway inside this method?

Reply:
Fixed with: https://github.com/wso2/product-microgateway/pull/2653/commits/058939d2175822f8d5f60ff47f239ba21d5640fa